sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/ApiClientConfigurationTest.java

```diff
@@ -52,6 +54,11 @@ class ApiClientConfigurationTest {
 
   private ClientTestApp app;
 
+  @BeforeAll
+  static void beforeAll() {
+    WireMock.configureFor("http", "localhost", WIRE.getPort());
```

Comment:
Do we need a comment here? At least I don't understand why this configuration is needed.

Reply:
Added comment
` //    Explicitly set the wiremock host and port, to keep configuration after reset in setUp()`

https://github.com/SDA-SE/sda-dropwizard-commons/pull/2973